sklego/preprocessing.py

```diff
@@ -420,3 +420,68 @@ def transform(self, X):
         X = check_array(X, estimator=self)
 
         return X @ self.inv_R_ / self.normalization_vector_
+
+
+def scaler_(vec, unto):
```

Comment:
You actually already compute the projection here, right? Should this function be called `scalar_projection`? and the `project_` then `vector_projection`?

Reply:
it'd be a minor rename only, but sure, it's more explicit that way.

Reply:
done.